src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCellModel.swift

```diff
@@ -23,24 +25,33 @@ class HomeStatisticsCellModel {
 			}
 			.store(in: &subscriptions)
 		
-		homeState.$selectedLocalStatistics
-			.sink { [weak self] selectedLocalStatistics in
-				  self?.selectedLocalStatistics = selectedLocalStatistics
-				  Log.debug("HomeState did update. \(private: "\(self?.selectedLocalStatistics.count ?? -1)")", log: .localStatistics)
+		localStatisticsProvider.regionStatisticsData
+			.sink { [weak self] regionStatisticsData in
+				  self?.regionStatisticsData = regionStatisticsData
+				  Log.debug("Updating local statistics cell model. \(private: "\(self?.regionStatisticsData.count ?? -1)")", log: .localStatistics)
 			}
 			.store(in: &subscriptions)
 	}
 
 	// MARK: - Internal
 
-	let homeState: HomeState
-
 	/// The default set of 'global' statistics for every user
 	@DidSetPublished private(set) var keyFigureCards = [SAP_Internal_Stats_KeyFigureCard]()
-	@DidSetPublished private(set) var selectedLocalStatistics = [SelectedLocalStatisticsTuple]()
+	@DidSetPublished private(set) var regionStatisticsData = [RegionStatisticsData]()
+
+	func addLocalStatistics(region: LocalStatisticsRegion) {
+		localStatisticsProvider.add(region)
+	}
+
+	func remove(_ region: LocalStatisticsRegion) {
```

Comment:
```suggestion
	func removeLocalStatistics(_ region: LocalStatisticsRegion) {
```

Reply:
Ah, I removed the inconsistency now, but decided the other way around and made the add method shorter